Tests/Unit/Subscribers/CronJobTest.php

```diff
@@ -104,16 +115,7 @@ public function test_export_dynamic_stream()
         $this->productSearch->expects($this->once())
             ->method('getProductFromConditionStream')
             ->with($stream)
-            ->willReturn(new ProductSearchResult(
-                [
-                    'SW100001' => new ListProduct(14, 26, 'SW100001'),
-                    'SW100003' => new ListProduct(15, 35, 'SW100002'),
-                ],
-                2,
-                [],
-                new Criteria(),
-                Shopware()->Container()->get('shopware_storefront.context_service')->getShopContext())
-            );
+            ->willReturn($ps);
```

Comment:
Keine Ahnung warum aber es wollte lokal einfach nicht laufen, wenn der Constructor in der Funktion aufgerufen wurde